backend/src/main/java/fk/prof/backend/model/assignment/AggregationWindowPlanner.java

```diff
@@ -71,6 +80,17 @@ public AggregationWindowPlanner(Vertx vertx,
     this.aggregationWindowDurationInSecs = aggregationWindowDurationInSecs;
     this.policyRefreshBufferInSecs = policyRefreshBufferInSecs;
 
+    processGroupTag = new ProcessGroupTag(processGroup.getAppId(), processGroup.getCluster(), processGroup.getProcName());
```

Comment:
Is it useful to have metric per process group. It becomes difficult to track if there are so many time series.